documentation/documentation/events/versioning/index.md

```diff
@@ -0,0 +1,58 @@
+<!--Title:Events Versioning-->
+<!--Url:versioning-->
+
+Events by their nature represent facts that happened in the past. They should be immutable even if they had wrong values (as we can only roughly guess what should be the missing property value).
+
+However, in practice, it's unavoidable in the living system to not have the event schema migrations. They might come from:
+
+- bug - eg. typo in the property name, missing event data, 
+- new business requirements - eg. besides storing the user email we'd like to be also storing its full name,
+- refactorings - eg. renaming event class, moving to different namespace or assembly,
+- etc.
+
+Depending on the concrete business case we may use a different technique for handling such event migrations.
+
+## Namespace migration
+
+Marten by default tries to find the event class based on the fully qualified assembly name (it's stored in `mt_dotnet_type` column of `mt_events` table, read more in <[linkto:documentation/events/schema;title=events schema documentation]>). 
+When it is not able to find event type with the same assembly, namespace and type name then it tries to make a lookup for mapping on the event type name (stored in `type` column of `mt_events` table).
+
+Such mapping needs to be defined manually:
+
+- either by registering events with store options `Events.AddEventTypes` method,
+- or by defining custom mapping with `Events.EventMappingFor` method.
+
+For the case of namespace migration, it's enough to use `AddEventTypes` method as it's generating mapping based on the event type. So eg. having `OrderStatusChanged` event changed from:
```

Comment:
Request for a small tweak to this sentence, change "So eg. having..." to "As an example, change `OrderStatusChanged` from:"